### ISSUE TITLE
Uncaught exceptions in model with lots of connections

### ISSUE THREAD

**Issue by ksmyth** [webgme/webgme#1545]
Import this project:
https://s3.amazonaws.com/tonka-installer/guest%2BADMEditor_metaOnly_14ea61.webgmex

then try:
http://127.0.0.1:8888/?project=guest%2BADMEditor_metaOnly&branch=master&node=%2FW%2Fr%2FT%2F4&visualizer=ModelEditor&tab=0&layout=DefaultLayout&selection=

```
6:8888/dist/webgme.2.19.0.dist.build.js:33 Uncaught TypeError: Cannot read property '0' of null
    at r._handleWorkerResponse (:8888/dist/webgme.2.19.0.dist.build.js:33)
2AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
6AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
3AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
2:8888/js/Widgets/DiagramDesigner/AutoRouter.Worker.js:93 Uncaught RangeError: Maximum call stack size exceeded
```

**Comment by pmeijer** [comment]
This is a strange issue. First of all it doesn't happen on FireFox, secondly when debugging it and stopping on caught exception the stack is practically empty. I found this issue which could be related:

https://github.com/blimmer/crypto-js/issues/109

**Comment by ksmyth** [comment]
The issue is happening inside native code, which is why the stack isn't long. The native code is the recursive structured clone algorithm:

`worker.postMessage(response);`

https://developer.mozilla.org/en-US/docs/Web/API/Worker/postMessage
https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm

Here's an example of response:

[response.txt](https://github.com/webgme/webgme/files/1604145/response.txt)

It is too heavily nested (over 400 levels of nesting):

    {
        "box": {
            "atomic": false,
            "childBoxes": [
                null,
                null,
                null
            ],
            "id": "BOX_000310",
            "owner": {
                "horizontal": {
                    "ishorizontal": true,
                    "orderFirst": {
                        "orderNext": {
                            "blockNext": null,
                            "orderNext": {
                                "blockNext": null,
                                "orderNext": {
                                    "blockNext": null,

**Comment by pmeijer** [comment]
Thanks Kevin that helps a lot!

**Comment by pmeijer** [comment]
Closed by #1566
